src/test/java/ListFindElements.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class ListFindElements extends BaseTest {
+
+  @Test
+  public void verifyEachElementInTheList() throws InterruptedException {
+
+    driver.get("https://deens.com/");
+    WebElement elem = driver.findElement(By.cssSelector("a[href*='fun-getaway-with-friends-in-london']"));
+    ((JavascriptExecutor) driver).executeScript("arguments[0].scrollIntoView(true);", elem);
```

Comment:
Are you sure that you need to use this scroll?

Reply:
The exercise was about going through the list and compare the title of each element in the list with the title on the page for each element after clicking the element.